pkg/wallet/api.go

```diff
@@ -90,6 +90,16 @@ type DIDCreator interface {
 	CreateDID(method string, opts ...DocOpts) (*did.Doc, error)
 }
 
+// KeyConverter provides methods for converting signing to encryption keys
+type KeyConverter interface {
+	// ConvertToEncryptionKey creates and persists a Curve25519 keypair created from the given SigningPubKey's
+	// Ed25519 keypair, returning the EncryptionPubKey for this new keypair.
+	ConvertToEncryptionKey(key string) (string, error)
+
+	// GetEncryptionPubKey converts the given SigningPubKey to an EncryptionPubKey
```

Comment:
Does this also convert? But not persist?

Reply:
Yes, I'll move it to cryptoutil

Reply:
Moved, along with a couple other static util functions.